src/utils/navigation.tsx

```diff
@@ -33,12 +33,12 @@ export function settingToRouteName(
 ): keyof TabNavigatorParams {
   switch (setting) {
     case 'assistant':
-      return 'Assistant';
+      return 'Dashboard';
     case 'departures':
       return 'Nearest';
     case 'ticketing':
       return 'Ticketing';
     default:
-      return 'Assistant';
```

Comment:
@gorandalum something is going on here after we removed the Assistant, I have replaced it with Dashboard the returns but not quite sure, could you please take a look?

Reply:
@gorandalum together with that, there is a compilation error in src/screens/AnonymousPurchase/ConsequencesScreen.tsx

Reply:
In the `ConsequencesScreen` you need to navigate to the `DashboardRoot` through both the `TabNavigator` and `Dashboard` first.

Reply:
Do you have any ideas on how to do it? I am a bit confused here.

Reply:
Something like this? It is written for hand, so might be some details that is wrong, but maybe enough to get the gist of it?
```
navigation.navigate("TabNavigator", {
  screen: 'Dashboard',
  params: {
    screen: 'DashboardRoot',
  }
})
```